ppa/archive/templates/archive/digitizedwork_detail.html

```diff
@@ -87,7 +102,13 @@ <h1 class="header">{{ object.title }}</h1>
             <tr class="source-link">
                 <th scope="row">{{ object.get_source_link_label }}</th>
                 <td>
+                    {% if object.source == "G" %}
```

Comment:
same problem with the check here — need to either use the constants on the class or methods on the class (if we have them, but I don't think we do and it doesn't make sense to add them)

Also, we should only use the page url if there are pages defined